docs/developer-docs/latest/setup-deployment-guides/configurations/required/databases.md

```diff
@@ -286,4 +286,21 @@ The following documentation covers how to install SQLite locally (for developmen
 
 :::caution
 Installation guides for other databases (MySQL, MariaDB) are being reworked. [Contributions](https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md) are most welcome.
-:::
\ No newline at end of file
+:::
+
+:::note
+When connecting Strapi to a postgres database, the database user requires SCHEMA permissions. While the database admin has this permission by default, a new database user that's created specifically for the Strapi application will not and result in a 500 error when trying to load the admin console.
```

Comment:
```suggestion
When connecting Strapi to a PostgreSQL database, the database user requires SCHEMA permissions. While the database admin has this permission by default, a new database user explicitly created for the Strapi application will not. This would result in a 500 error when trying to load the admin console.
```